packages/tldraw/src/lib/tools/SelectTool/childStates/PointingArrowLabel.ts

```diff
@@ -100,14 +102,24 @@ export class PointingArrowLabel extends StateNode {
 			nextLabelPosition = 0.5
 		}
 
+		this.didDrag = true
```

Comment:
1. you might be able to use `editor.inputs.isDragging` here
2. I'm not sure i understand how this works. we check `didDrag` in `onPointerUp`, but we set it in `onExit`. I would expect `onPointerUp` to be called before `onExit` though, because after on exit we're no longer in this state node and it shouldn't be receiving events. any ideas what's going on here?